examples/dev/Dashboard.js

```diff
@@ -58,7 +58,7 @@ module.exports = () => {
       username: 'John',
       license: 'Creative Commons',
     },
-    restrictions: { requiredMetaFields: ['caption'] },
```

Comment:
Shouldn't we keep this line but commented? It could be useful to have a way to quickly adding restrictions.